### ISSUE TITLE
[BUG] Cards No Longer Render in AnkiWeb

### ISSUE THREAD

**Issue by tahoeschrader** [migaku-official/Migaku-Kanji-Addon#196]
**Describe the bug**
A couple of days ago, cards stop being able to render in AnkiWeb. AnkiWeb might gone through some upgrades recently that broke something because these cards used to render for me. 

I have tried this in multiple browsers and on my phone. 

**To Reproduce**
1. Create a deck with KanjiGOD cards
2. Go to AnkiWeb
3. Try to review them 

**Expected behavior**
Front and back of KanjiGOD cards should render in AnkiWeb. 

**Desktop (please complete the following information):**
N/A

**Additional context**
N/A

**Screenshots**
![image](https://github.com/migaku-official/Migaku-Kanji-Addon/assets/26147461/b0555fcd-dbaa-4a87-9d6b-e2468d26b7ab)

**Comment by tahoeschrader** [comment]
I was looking at the Chrome dev tools and it showed a CORS missing header error on the kanji font .tff files that's causing them to be blocked. I'm wondering if it has something to do with that?

**Comment by mjuhanne** [comment]
<img width="1432" alt="Screenshot 2023-08-23 at 16 12 13" src="https://github.com/migaku-official/Migaku-Kanji-Addon/assets/12277227/7f46b59d-c655-4e4f-865e-73aca32e4c1c">

I can confirm this. I see the font loading error as well but the main thing that that prevents this to function correctly is that it cannot find/use jQuery ($-function related stuff). jQuery is supposed to be there at AnkiWeb (it's not included by add-on).  @hgiesel @MinmoTech  Any ideas? Some change in AnkiWeb broke this?

**Comment by hgiesel** [comment]
Hm, other cards have access to jquery (`$`) just fine. It seems more likely to me, that some other issue cause the scripts to stop executing, which then leads to jquery not being initialized.

**Comment by mjuhanne** [comment]
I made a test deck with very simple template which works ok on Anki Desktop but fails in AnkiWeb. 

```
<div>question jQuery: <span id="question"></span></div>
<div>question JavaScript: <span id="question2"></span></div>
<script>

$('#question').html("{{ Kanji }}");

document.getElementById("question2").innerHTML = " {{ Kanji }} ";

</script>
```

So definitely not a problem with Migaku but with something else there..

**Comment by tahoeschrader** [comment]
I've been using V3 scheduler for a couple months on Anki Desktop though and only saw the cards not working a couple days ago.

**Comment by tahoeschrader** [comment]
See response from dae over at Anki:

https://forums.ankiweb.net/t/jquery-seems-to-be-broken-after-v3-updates-to-website/33575

They suggest adding 

```
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
```

to the top of the card template, or moving away from relying on JQuery.

**Comment by mjuhanne** [comment]
Ah, that's unfortunate that they have decided to remove jQuery without any heads-up or early warnings.

I tried adding the jQuery link as adviced, but it still complains about $-references when processing the front side of the FIRST card in the deck. Back side of the first card and then subsequent cards seem to work ok.  

Then I converted the jQuery to vanilla JS on front-production/recognition.html files and removed the link to jQuery there (so jQuery is referenced only on back side htmls).  This time, the back side cards all fail due to jQuery not working.   Is the jQuery not ready immediately after it is loaded?   

Converting back side HTMLs to vanilla JS is going to be annoying (over 200 jQuery $-operators found...)

**Comment by tahoeschrader** [comment]
Yeah it's pretty frustrating. AnkiWeb is the only way I can rep these cards when I'm not at home and Migaku has given no inclination as to when this feature will be added to Migaku Memory, if ever...

I ran into the same first card failure thing when I tried adding that JQuery initialize line as well but I also sadly have zero JavaScript or JQuery or anything web-related experience to dig much further without some guidance.

**Comment by mjuhanne** [comment]
@tahoeschrader 
If you want to try, [In this branch](https://github.com/mjuhanne/Migaku-Kanji-Addon/tree/no-jquery-2) are the few corrections I needed to get AnkiWeb running.

I don't personally use AnkiWeb and the test branch I use everyday is way ahead of the official Migaku branch, but let me know if you run into problems.  If his works ok, I should put up a PR so others will be able to use AnkiWeb as well

**Comment by hgiesel** [comment]
@mjuhanne 

Scripts by default are executed in "load-order", that means that scripts which have to fetch  their content first (`src` attribute), will always be executed after scripts that are inline.
By simply inserting jquery into the template, we circumvent that.

**Comment by tahoeschrader** [comment]
@hgiesel 

Just tried out your code and it works for me!

Back:
<img width="1133" alt="Screenshot 2023-08-28 at 11 14 19 PM" src="https://github.com/migaku-official/Migaku-Kanji-Addon/assets/26147461/2de56770-15e4-4946-9d42-38c6eb74560a">

Front:
![image](https://github.com/migaku-official/Migaku-Kanji-Addon/assets/26147461/0a0e3037-823e-4c3a-84f3-653fd0bc12d9)
